app/views/shared/_get_started_banner.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="slab slab--cta">
+  <div class="grid">
+    <div class="grid__item width-one-whole text--centered">
+      <div class="h2 spacing-below-35"><%=t("views.public_pages.home.file.header") %></div>
+      <% if enable_online_intake? %>
+        <%= link_to question_path(id: QuestionNavigation.first),
+          class: "button button--wide button--primary button--centered spacing-below-0",
+          **mixpanel_info(local_assigns) do %>
```

Comment:
`**mixpanel_info(local_assigns)` is a very abstract and confusing what to say "track clicks on this link with this data". it's far too abstract.